services/drone/drone-build.service.js

```diff
@@ -0,0 +1,131 @@
+'use strict'
+
+const Joi = require('joi')
+const serverSecrets = require('../../lib/server-secrets')
+const { isBuildStatus, renderBuildStatusBadge } = require('../build-status')
+const { optionalUrl } = require('../validators')
+const { BaseJsonService } = require('..')
+
+const DroneBuildSchema = Joi.object({
+  status: Joi.alternatives()
+    .try(isBuildStatus, Joi.equal('none'))
+    .required(),
+}).required()
+
+const queryParamSchema = Joi.object({
+  server: optionalUrl,
+}).required()
+
+module.exports = class DroneBuild extends BaseJsonService {
+  static get category() {
+    return 'build'
+  }
+
+  static get route() {
+    return {
+      queryParamSchema,
+      base: 'drone/build',
+      pattern: ':user/:repo/:branch*',
+    }
+  }
+
+  static get staticPreview() {
```

Comment:
Think we can remove this function entirely right?

Reply:
Yes, I think I copied it from Travis. It _was_ used in the examples, but not anymore. My bad for not removing this earlier.

Reply:
Done!